File: backend/src/database/seeders/CategorySeeder.php
<?php

namespace Database\Seeders;

use App\Models\Category;
use Illuminate\Database\Seeder;

class CategorySeeder extends Seeder
{
    /**
     * Run the database seeds.
     */
    public function run(): void
    {
        $categories = [
            [
                'name' => 'Cobre',
                'description' => 'Filamento de cobre',
            ],
            [
                'name' => 'Cuerda',
                'description' => 'Filamentos de cobre ya tejidos',
            ],
            [
                'name' => 'PVC',
                'description' => 'Policlorulo de vinilo',
            ],
            [
                'name' => 'Master',
                'description' => 'Master',
            ]
        ];

        foreach ($categories as $category) {
            Category::create($category);
        }
    }
}

File: backend-dotnet/src/Ikhana.Infrastructure/Data/Seeders/CategorySeeder.cs
using Ikhana.Application.Common.Interfaces;
using Ikhana.Domain.Entities;
using Ikhana.Infrastructure.Persistence;
using Microsoft.EntityFrameworkCore;

namespace Ikhana.Infrastructure.Data.Seeders;

public class CategorySeeder : IDataSeeder
{
    private readonly AppDbContext _context;

    public int Order => 3;

    public CategorySeeder(AppDbContext context)
    {
        _context = context;
    }

    public async Task SeedAsync(CancellationToken cancellationToken = default)
    {
        if (await _context.Set<Category>().AnyAsync(cancellationToken))
            return;

        var categories = new (string Name, string Description)[]
        {
            ("Cobre", "Filamento de cobre"),
            ("Cuerda", "Filamentos de cobre ya tejidos"),
            ("PVC", "Policlorulo de vinilo"),
            ("Master", "Master"),
        };

        foreach (var (name, description) in categories)
        {
            _context.Set<Category>().Add(new Category
            {
                Name = name,
                Description = description,
            });
        }

        await _context.SaveChangesAsync(cancellationToken);
    }
}
